main_doc.md

```diff
@@ -111,7 +111,8 @@ _(仅限本科新生与本科转学生)_ [New Student Checklist](http://www.chec
 * 如果国际生新生 Orientation 只显示5到7月的日期，请发邮件联系对应学院的新生联络邮箱来选择8月的 Orientation 日期。在第一次做出选择后，Orientation 的日期后续 **可以** 在 Checklist 里更改。
 
 * Placement Test
-    * Placement test 中<font color="red">数学必考</font>，分配结果（课号）MATH1241/1271/1371 (Calculus I) 为该测试的最高水平。
+    * Placement test 中<font color="red">数学必考<sup>*</sup></font>，分配结果（课号）MATH1241/1271/1371 (Calculus I) 为该测试的最高水平。
```

Comment:
We don't have to use <sup>*</sup> and  * to annotate the following content

Markdown's built-in intent should be enough for the following details.